docs/overview.rst

```diff
@@ -44,6 +44,8 @@ These computers should all physically be in your organization's office.
 
 .. _`Tails operating system`: https://tails.boum.org
 
+.. _securedrop_architecture:
```

Comment:
Completely agree. Amended 139a76ed5d9d40f93fbe95e4094423226d3c6c41 as f4bd6e8ba57066d1c1f28ab3e416e60e50374669 to include the update.